src/bitwarden_exporter/__main__.py

```
@@ -27,6 +27,47 @@
+    elif secret_path.startswith("file:"):
+        secret_path = secret_path[5:]
+        if not os.path.exists(secret_path):
+            raise FileNotFoundError(f"File not found: {secret_path}")
+        if not os.path.isfile(secret_path):
+            raise ValueError(f"File is not a file: {secret_path}")
+    if secret_path.startswith("jmespath:"):
+        jmespath_expression = secret_path[len("jmespath:") :]
+        jmespath_password = jmespath.search(jmespath_expression, all_items_list)
+
+        if not jmespath_password:
+            raise BitwardenException("Vault password is not found")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)